docs/reference/migration/migrate_2_0.asciidoc

```diff
@@ -384,6 +384,30 @@ http.cors.enabled: true
 http.cors.allow-origin: /https?:\/\/localhost(:[0-9]+)?/
 ---------------
 
+=== Expressions API changes
+
+When using expressions the .value member variable has been removed in favor of just the
+field name since expressions do not support multi-valued fields.
+
+Example:
+1.x -- doc['fieldname'].value
+2.0 == doc['fieldname']
+
+Expressions also allow member methods with no arguments to used.  The following member
```

Comment:
Actually the "new" part of this should move to expression docs.  See `scripting.asciidoc`